tools/sidekick/config.json

```diff
@@ -32,6 +32,31 @@
       "excludePaths": [ "/**" ],
       "includePaths": [ "**/:x**" ]
     },
+    {
+      "containerId": "tools",
+      "title": "Send to CaaS",
+      "id": "sendtocaas",
+      "environments": [
+        "dev",
```

Comment:
what are these environments referring to? I think we only need prod.

Reply:
I've updated the PR https://github.com/adobecom/dme-partners/pull/7/commits/9aa720c9513ad3825c29b0e970530d12457b1686

Reply:
Actually, I think they refer to the environments where 'Send to CaaS' option will be available. It makes sense to me to keep 'live' & 'preview' environments (adobecom.hlx.live & adobecom.hlx.page) for now for testing purposes. Wdyt?